doc/src/manual/modules.md

```diff
@@ -168,6 +168,23 @@ Importantly, the module name `NiceStuff` will *not* be in the namespace. If you
 julia> using .NiceStuff: nice, DOG, NiceStuff
 ```
 
+!!! note
+    Qualifying the names being used as in `using NiceStuff: NiceStuff, nice` is recommended over plain
+    `using Foo` for released packages, and other code which is meant to be re-used in the future with
```

Comment:
A third name, `Foo`... Let's stick with one, e.g. `NiceStuff` perhaps?
```suggestion
    `using NiceStuff` for released packages, and other code which is meant to be re-used in the future with
```

Reply:
Right, yeah this naming conflict should be avoided. I think for the purposes of this `!!! note` though, maybe it'd be better to use a new set of names `Foo` and `Bar` because we want to talk about conflicts, and we also want this `!!! note` section to be readable standalone.